### PR TITLE
fix: remove applySmooth in button

### PR DESCRIPTION
**Pull request by joshhowenstine**
## Description

It has been observed that when a button is displayed as the first item in a Row or Column and immediately receives focus it sill not display the text or icon for the first item. 

button

[Does not work if gains focus right away before this PR](https://lightningjs.io/playground/#N4IgJgpgDhB2mwMYEsIGcQC5QAEA2yA5gBYAusyshAVmgPSID2AThFiGaVGpnQ2LAB0aANYBPKAENEIwZABudfETIUqtBiwg4ATIICM+wQAYQAGhDKS5SjXpowI9p269+Q0ROmyFSgtbU7OgcRHAAWQQAOQQBmc0t-VVsNAFdkAFomAFsoRlg4UgxMDlIuHj5EAWFxKRk5CEUrJPV6NMzGHLyCjABfCzREZmQoQqxQPEkUpGJBWjGQADNkPDZiianEGbmLJkh2AB1YAAIT05PkTuZSI4BBKCgjheYOo4ByQTo7qFm0V4BuQ5nM6AoEnADulDAjDBgkkpWkxAAkrA0DBEKQWAAKCHwaGCPBUACUANgINBTBR10YI2QeTQRwAvEdgGhSJJCBBMMywVz9ABOHTGMxHYi84yRIVHRArSTMADCjDwLC5xgAHsYNZqNT0eiTQScKayjpJ7oyjvkwbd7pjqaRaSjiWSgVDECksgVBAAjRhgMSw+5wMBy4jLMCYk3fVnsiCCDmkOWSWDySRoTGEx3HU7xSAweBwFDoLAAbRAHy+PxAAF0+iAABIQPBKgDqLDwYAr2EWy1WdYbzdb7e2IF2PYuuSuRwAMiobFRHs8sm8mrOgkxWP9DmOWNdgEcAErQ4UAIRSpTyRx685er2XgVSGWyuXysEKr0OhwgqvH10gC0meGuaUU3petG0YFtmDbI5P1IQN6WnAJbEEBVOmfHcnSjO1ECOAB9WCcgmWC02ZJ0zlYUgUmYY5gFI-UTzPWB9C5GjM31fVSAkTkjnojFYDMWi2JOO1SBWLlXh4899FefjWMEs5VS5QUhQEwSxEUjUZLkk4+hUs4JNgHRmN09jOK5fTNK0s5hNEt59KOHRpOM0EFPsjSnKBNSjhiMJlNktiehUgLZKCjC2Sw3DMPQYiWLY8jKOOIt3KAtB6QAMWQZgjRguCjlIEN6RiyzThwuNUsYV00AgMNCRIvyipOOKqNy-LBE2UNp1ZQQlky0h3LOILLJ0uryQmFKpxTa5svgek8uQAq+uK0rypSSrqtq+rQUa45ZrQVqQzbDrSHxCaFu09yBv1SsnRC2ScMNUhmBSdFopUnbBBwyrSAAZTCiBMTeyLGQZJlXnSnrXiOAB+N5JwmiGxLB1lXkJFTPuRWDmGTPBMWIhkAD51q0t6Pogb7fv+lrAeBkHEdICHodeWGkaOBGMqRlHhovYUlOMDn+qdQ4emzaBA3zVAihLW9knoNc2AsKWWjoNpHy6F8MGrCxyzmTslhWdgtYwHYfVHS5rgQ5pCCvRcb0SFcNFljdSQoU2jlA-tILAK23g+N3wIHH5HY-L9tyOX9-0A0b6S+aDVVg6apxnQJBC+AhEDhe1CZOTDkGwvCIAIuE-pqwrNtJ+LM8E32ILbZiOJgLkq4HC7QWbi931upaKqql7Oa25q5sENlCExV5G495HrsF4XcwQAsJYSRPpc0Vh4h9vs-Y9itqxrNlPSKEBJEbEAeiAA)

[Works with timeout before this PR](https://lightningjs.io/playground/#N4IgJgpgDhB2mwMYEsIGcQC5QAEA2yA5gBYAusyshAVmgPSID2AThFiGaVGpnQ2LAB0aANYBPKAENEIwZABudfETIUqtBiwg4ATIICM+wQAYQAGhDKS5SjXpowI9p269+Q0ROmyFSgtbU7OgcRHAAWQQAOQQBmc0t-VVsNAFdkAFomAFsoRlg4UgxMDlIuHj5EAWFxKRk5CEUrJPV6NMzGHLyCjABfCzREZmQoQqxQPEkUpGJBWjGQADNkPDZiianEGbmLJkh2AB1YAAIT05PkTuZSI4BBKCgjheYOo4ByQTo7qFm0V4BuQ5nM6AoEnADulDAjDBgkkpWkxAAkrA0DBEKQWAAKCHwaGCPBUACUANgINBTBR10YI2QeTQRwAvEdgGhSJJCBBMMywVz9ABOHTGMxHYi84yRIVHRArSTMADCjDwLC5xgAHsYNZqNT0eiTQScKayjpJ7oyjvkwbd7pjqaRaSjiWSgVDECksgVBAAjRhgMSw+5wMBy4jLMCYk3fVnsiCCDmkOWSWDySRoTGEx3HU7xSAweBwFDoLAAbRAHy+PxAAF0+iAABIQPBKgDqLDwYAr2EWy1WdYbzdb7e2IF2PYuuSuRwAMiobFRHs8sm8mrOgkxWP9DmOWNdgEcAErQ4UAIRSpTyRx685er2XgVSGWyuXysEKr0OhwgqvH10gC0meGuaUU3petG0YFtmDbI5P1IQN6WnAJbEEBVOmfHcnSjO1ECOAB9WCcgmWC02ZJ0zlYUgUmYY5gFI-UTzPWB9C5GjM31fVSAkTkjnojFYDMWi2JOO1SBWLlXh4899FefjWMEs5VS5QUhQEwSxEUjUZLkk4+hUs4JNgHRmN09jOK5fTNK0s5hNEt59KOHRpOM0EFPsjSnKBNSjhiMJlNktiehUgLZKCjC2Sw3DMPQYiWLY8jKOOIt3KAtB6QAMWQZgjRguCjlIEN6RiyzThwuNUsYV00AgMNCRIvyipOOKqNy-LBE2UNp1ZQQlky0h3LOILLJ0uryQmFKpxTa5svgek8uQAq+uK0rypSSrqtq+rQUa45ZrQVqQzbDrSHxCaFu09yBv1SsnRC2ScMNUhmBSdFopUyrSGRWDmGTPBMWIhkAD51q0nbBBwt6AGUwogTEQcixkGSZV50p614jgAfjeScJtRsTkdZV5CUC4UlOMQngqdQ4emzaBA3zVAihLW9knoNc2AsJmWjoNpHy6F8MGrCxyzmTslhWdghYwHYfVHS5rgQ5pCCvRcb0SFcNFZjdSQoWWjlA-tILAJW3g+PXwIHH5NY-L9tyOX9-0A0b6S+aDVVg6apxnQJBC+AhEDhe0gZOTDkGwvCIAIuFoZqwrNogCimpjwTTYgttmI4mAuWTgcLtBHOL3fW6loqqqXuGo4tuaubBDZQhMVeLODYJ67Kep3MEALBmEk95nNFYeITb7M2DYrasazZT0ihASRGxAHogA)

## References

[JIRA Ticket](https://ccp.sys.comcast.net/browse/LUI-1163)

## Testing

This currently was not reproduced in Storybook without a lot of changes. Unfortunatly we might need to test this in flex or another consuming app manually by building an artifact.

## Automation

Should not require any changes

### CODE REVIEW

File: packages/@lightningjs/ui-components/src/mixins/withThemeStyles/index.js
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/mixins/withThemeStyles/index.js:1]: Does it make sense to pull this out into a separate PR? It feels disconnected from the Button fix.

**Review by ImCoolNowRight — APPROVED** [review]
I pulled this into IS and it seems to have fixed the issue with NavigationLeft where we had to force the first NavItem to update to get the icon loading properly. So I think this is good!

File: packages/@lightningjs/ui-components/src/mixins/withThemeStyles/index.js
Comment (joshhowenstine) [reply packages/@lightningjs/ui-components/src/mixins/withThemeStyles/index.js:1]: Yeah, I agree. I think that is a good idea. Removed.

**Review by anthony9187 — APPROVED** [review]

**Review by soumyaloka — APPROVED** [review]
